### DISCUSSION TITLE
Minor 3.1.0

### DISCUSSION THREAD

**Discussion by XFox111** [XFox111/PasswordGeneratorExtension#309] [General]
> ## 🚀 Patch Tuesday update
> This release is a part of our new initiative!
From now on we are starting to roll out updates on every first Tuesday of the month, which will include bugfixes, security and dependency updates to keep the project's security and stability up to date!

## What's new
- Added ability to change min/max values of quick adjustment length slider
- Added more customization options to quick adjustment panel (include/exclude characters)
- Quick adjustment panel now collapses if settings section is expanded
- Updated Brazilian Portuguese locale (thanks to @maisondasilva)
- Dependency updates and security patches

### Fixed security issues in this update
- CVE-2024-23331

See full change log on #308

Refer to [Download section of the README.md](https://github.com/XFox111/PasswordGeneratorExtension#download) for sideloading instructions and download links.

<hr /><em>This discussion was created from the release <a href='https://github.com/XFox111/PasswordGeneratorExtension/releases/tag/v3.1.0'>Minor 3.1.0</a>.</em>

**Comment by maisondasilva** [comment]
Great job!

**Comment by maisondasilva** [comment]
@XFox111 update image [README.md](https://github.com/XFox111/PasswordGeneratorExtension/blob/main/README.md) for new version!

**Reply by XFox111** [reply]
Yeah, that would be great. Though I think I'll leave it until the next update cycle - I have one more feature planned which will require to update promotional materials as well. So, I think it will be better to make it in one go instead.
